apple/Sources/FerrostarSwiftUI/Views/InstructionsView.swift

```diff
@@ -24,72 +32,110 @@ public struct InstructionsView: View {
     ///   - visualInstruction: The visual instruction to display.
     ///   - distanceFormatter: The formatter which controls distance localization.
     ///   - distanceToNextManeuver: The distance remaining for the step.
+    ///   - remainingSteps: All steps remaining in the route, including the current step
     ///   - primaryRowTheme: The theme for the primary instruction.
     ///   - secondaryRowTheme: The theme for the secondary instruction.
-    ///   - showPillControl: If true, shows a pill control (to indicate an action/expansion).
+    ///   - isExpanded: Whether the instruction view starts expanded
+    ///   - sizeWhenNotExpanded: Size of the InstructionsView when minimized - useful for allocating space for this view
```

Comment:
I wish SwiftUI had a better model for this sort of thing. It's technically a binding, but really its sole purpose is _publishing_ values back out to observers. Maybe we can clarify this a bit in the comment (if I'm understanding it correctly)?

Reply:
Yes, you are understanding correctly — this is a `Binding` because we want the **child** component to communicate something to its **parent**. I think your additional documentation makes this clearer. 👍